unit/src/DiscreteFissionPDFTest.C

```diff
@@ -180,45 +177,26 @@ DiscreteFissionPDFTest::sampleFissionPKA()
   auto maxerror = std::max_element(error.begin(), error.end());
 
   //check if total energy is conserved
-  CPPUNIT_ASSERT( std::abs(total_energy / max - true_energy) < 1.0 );
+  ASSERT_TRUE(std::abs(total_energy / max - true_energy) < 1.0);
```

Comment:
You can augment assertions like this to add custom output when they fail:

```c++
ASSERT_TRUE(false) << "oh no - how could false be not true?";
```
...(in case you weren't aware already).

Also, I usually use ``EXPECT_[bla]`` instead of ``ASSERT_[bla]`` because the latter immediately terminates execution of everything in the ``TEST[_F]`` function while the former allows you to collect output on all of the checks before terminating.  Use whatever you think is most appropriate.

Reply:
Thanks, Robert, I'll make the changes accordingly!